cogs/utility.py

```diff
@@ -59,6 +61,21 @@ async def serverinfo(self, ctx):
         embed.add_field(name="Features", value=features)
         await ctx.respond(embed=embed)
 
+    @slash_command(brief="Get all server stickers & emojis!")
+    @discord.default_permissions(manage_guild=True)
+    @commands.cooldown(1, 600, commands.BucketType.user)
+    async def emoji_downloader(self, ctx):
+        await ctx.defer()
+        with zipfile.ZipFile('emoji_and_stickers.zip', 'w') as zipped_f:
+            for emoji in ctx.guild.emojis:
+                zipped_f.writestr(emoji.name + emoji.url[-4:], await emoji.read())
+            for sticker in await ctx.guild.fetch_stickers():
+                async with aiohttp.ClientSession() as session:
+                    async with session.get(url=sticker.url) as response:
+                        zipped_f.writestr(sticker.name + ".png", await response.read())
```

Comment:
You append a .png here, but what if the sticker is animated?

Reply:
There is no specific .apng extension, APNG is basically built into PNG, just like .gif supports both animated and static images